.circleci/config.yml

```diff
@@ -35,3 +35,33 @@ jobs:
       - run:
           name: Run integration tests
           command: ./scripts/circleci/run_automation_test.sh -t integration
+  publish:
+    docker:
+      - image: circleci/node:10.16.3
+    steps:
+      - checkout
+      - run:
+          name: Package
+          command: |
+            npm install cpx
+            npx lerna exec -- 'cpx package.json build/'
+            npx lerna exec -- 'cpx README.md build/'
+            npx lerna exec -- 'cpx ../../LICENSE build/'
+            npx lerna exec -- 'cd build'
```

Comment:
I think this line is redundant: ` npx lerna exec -- 'cd build'`

Reply:
Removed it